.github/workflows/pr_push.yml

```diff
@@ -48,6 +48,31 @@ jobs:
         - name: Build
           run: >
             cmake --build ${{github.workspace}}/build --config Release -j ${{matrix.nproc}}
+
+    CodeStyle:
+      name: Coding style
+      runs-on: ubuntu-latest
+
+      steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install apt packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake clang-format-15
+
+      - name: Configure CMake
+        run: >
+          cmake
+          -B ${{github.workspace}}/build
+          -DUMF_FORMAT_CODE_STYLE=ON
```

Comment:
The coding style check is so fast that - in my opinion - (apart from this build at the beginning that no doubt is needed) the coding style check could be added to all basic builds, just to be sure that coding style is verified in all possible configurations.

Reply:
The coding style is checked for all h, hpp, c, cpp etc files in the project. There is no need to run it multiple times. If we decided to use separate workflows we should keep them short and  ideally each should be responsible for something different.

Reply:
Right